mitodl/open-discussions
# Implement adding and removing moderators and contributors

#### What are the relevant tickets?
Fixes #864 

There is a question of pagination, not sure if that's a requirement for closing the issue

#### What's this PR do?
Adds a UI  to add and remove contributors and moderators

#### How should this be manually tested?
Go to the moderators tab and try adding and removing contributors via email address. If the add fails you should get a simple validation error. If the input is blank you should see a validation error about that. If you add a user correctly (even if they already exist) it should clear the field and the list should be updated.
